lib/qesapdeployment.pm

```diff
@@ -1996,6 +1996,34 @@ sub qesap_az_vnet_peering_delete {
     record_soft_failure("Peering destruction FAIL: There may be leftover peering connections, please check - jsc#7487");
 }
 
+=head3 qesap_az_peering_list_cmd
+
+    Compose the azure peering list command, using the provided:
+    - resource group, and
+    - vnet
+    Returns the command string to be run.
+
+=over 2
+
+=item B<RESOURCE_GROUP> - resource group connected to the peering
+
+=item B<VNET> - vnet connected to the peering
+
+=back
+=cut
+
+sub qesap_az_peering_list_cmd {
```

Comment:
Idea for some next PR
I like order in the words within `qesap_az_peering_list_cmd` more that what we did in `qesap_az_get_peering_name` and `qesap_az_get_active_peerings`

We could eventually think about renaming them to 
`qesap_az_peering_get_name` and `qesap_az_peering_get_active`